gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -91,6 +92,14 @@ def add_arguments(self, parser):
               'variants not defined in the header sections, or the definition '
               'of the the header fields do not match the field values.'))
 
+  def validate(self, parsed_args):
+    # type: (argparse.Namespace) -> None
+    if parsed_args.infer_headers and parsed_args.representative_header_file:
+      logging.warning('Both --infer_headers and --representative_header_file '
```

Comment:
I think we should just fail and force the user to choose the right flags. Users rarely look at warning logs :)

Reply:
Thanks Asha! It makes sense. I also have this concern. When I was testing I had to keep my eyes open since the warning was overwhelmed by lots of info logs.